ksim/task/rl.py

```diff
@@ -1944,7 +1944,8 @@ def _save(
         state: xax.State,
     ) -> None:
         model = eqx.combine(carry.shared_state.model_arrs, constants.constants.model_statics)
-        self.save_checkpoint(models=[model], optimizers=constants.optimizer, opt_states=carry.opt_state, state=state)
+        models = list(model) if isinstance(model, tuple) else [model]
```

Comment:
why this change? i think `model` might refer to the wrong thing - we aren't actually updating `model`, we're updating `carry`